gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_2_0/schema.yml

```diff
@@ -35,3 +35,9 @@ databaseChangeLog:
         - dropTable:
             cascadeConstraints: true
             tableName: ${gravitee_prefix}environment_domain_restrictions
+
+        - addColumn:
+            tableName: ${gravitee_prefix}quality_rules
+            columns:
+              - column: { name: reference_type, type: nvarchar(64), constraints: { nullable: false }, defaultValue: '' }
```

Comment:
64 chars are a bit too much imho but I cannot see any rules regarding reference_type, sometime we have 32 sometimes 64